Parser/FormTypeParser.php

```diff
@@ -52,9 +62,57 @@ class FormTypeParser implements ParserInterface
         'file'      => DataTypes::FILE,
     );
 
-    public function __construct(FormFactoryInterface $formFactory)
+    /**
+     * @var array
+     */
+    protected $extendedMapTypes = array(
+        DataTypes::STRING => array(
+            'text',
+            'Symfony\Component\Form\Extension\Core\Type\TextType',
```

Comment:
Please change to
use Symfony\Component\Form\Extension\Core\Type;

Type\TextType::class